management/baremetal.md

```diff
@@ -15,24 +16,24 @@ This document goes through the steps to prepare a baremetal or personal VM setup
 - a ssh key has been generated for `cluster-admin`. You can use `ssh-keygen` tool for this.
 - the public key for `cluster-admin` user is added to all other hosts in your setup. You can use `ssh-copy-id cluster-admin@<hostname>` for this, where `<hostname>` is name of the host in your setup where `cluster-admin` is being added as authorized user.
 
-###1. download and install cluster manager on the control host
+###1. Download and install Contiv Cluster
 ```
 # Login as `cluster-admin` user before running following commands
 git clone https://github.com/contiv/ansible.git
 cd ansible
 
-# Create inventory file
+# Create an inventory file
 echo [cluster-control] > /tmp/hosts
 echo node1 ansible_host=127.0.0.1 >> /tmp/hosts
 
-# Install cluster manager
+# Install Contiv Cluster
```

Comment:
This should be `Cluster Manager` or `clusterm service` as this playbook will only install and start clusterm on that host. `Contiv Cluster` might imply that even plugins, scheduler etc are also installed in this step which is not accurate.